catapult/utils.py

```diff
@@ -173,7 +173,12 @@ def to_human(data: Any):
 
 @to_human.register(list)
 def _(data: list):
-    return "\n".join(to_human(v) for v in data)
+    text = ", ".join(to_human(v) for v in data)
+    length = len(data)
+    if len(text) > 100:
+        return text[:97] + f"... ({length})"
```

Comment:
This means that these lines will end up being longer than 100 characters.

How about something like
```python
max_len = 100
if len(text) > max_len:
    trunc = f"... ({length})"
    return text[:max_len - len(trunc)] + trunc
```